### PR TITLE
Trigger inline styles on any character

### LINKED ISSUE #61: Trigger inline styles on any character
Dropbox Paper triggers inline styles on any character, not just space:

We should try doing that maybe!

### PR DESCRIPTION
**Pull request by mxstbr**
**TRY THE DEMO: https://public-lcgcixilzw.now.sh**

Closes #61

### CODE REVIEW

**Comment by brianlovin** [comment]
Might be some bugs still: https://www.dropbox.com/s/mewmgzxv1i6xk3d/draft-on-char.mov?dl=0

File: src/modifiers/__test__/handleInlineStyle-test.js [hunk 1]
```diff
@@ -151,7 +151,52 @@ describe("handleInlineStyle", () => {
...
+            data: {},
+          },
+        ],
+      },
+      after: {
+        entityMap: {},
+        blocks: [
+          {
+            key: "item1",
+            text: "hello inlinea style",
```
Comment (lachlanjc) [inline src/modifiers/__test__/handleInlineStyle-test.js:174]: You probably mean "inline" here :)

File: src/modifiers/__test__/handleInlineStyle-test.js [hunk 2]
```diff
@@ -426,6 +471,51 @@ describe("handleInlineStyle", () => {
...
```
Comment (lachlanjc) [inline src/modifiers/__test__/handleInlineStyle-test.js]: (same issue here)

File: src/modifiers/__test__/handleInlineStyle-test.js (hunk 1 above)
Comment (mxstbr) [reply src/modifiers/__test__/handleInlineStyle-test.js:174]: Actually, I didn't! This is how this test works:

Text before: (`|` is the cursor position)

```
hello inline| style
```

Then I press the "a" button, what do you expect to see? Exactly:

```
hello inlinea| style
```

Comment (lachlanjc) [reply src/modifiers/__test__/handleInlineStyle-test.js:174]: oh! sorry. well I've just learned something new. 🌟

**Comment by mxstbr** [comment]
Replaced by #73

**Pull request closed without merging**